### ISSUE TITLE
error - ReferenceError: EventTarget is not defined

### ISSUE THREAD

**Issue by Jairwin-L** [u-wave/react-vimeo#245]
install using pnpm, then pnpm dev
<img width="956" alt="image" src="https://github.com/u-wave/react-vimeo/assets/32570960/1171e355-bf9e-430b-8263-0d94db7ddacb">
env info
```bash
"@u-wave/react-vimeo": "^0.9.10",
"react": "18.2.0",
"react-dom": "18.2.0",
```

**Comment by goto-bus-stop** [comment]
It looks like you're server-rendering. this component does not support that.

**Comment by Jairwin-L** [comment]
> It looks like you're server-rendering. this component does not support that.

yes, but using yarn is ok, so I want to know why there is a problem with using pnpm

**Comment by goto-bus-stop** [comment]
that's impossible to say without a full, runnable example of the problem :)

**Comment by Jairwin-L** [comment]
let me try

**Comment by Jairwin-L** [comment]
@goto-bus-stop I re-reproduced this problem. It seems to be because of the node version. There are two node versions on my computer: 14.19.3 and 18.0.0. The 14 version will have this error, and the 18 version will not

**Comment by goto-bus-stop** [comment]
so really `@u-wave/react-vimeo` shouldn't be loaded at all on the server, as it includes `@vimeo/player` which is a client-side only library. This component can't do anything on the server and does not support server-side rendering.

**Comment by goto-bus-stop** [comment]
If you are using `next` I guess you should only import it in client components, but I don't use next myself so I can't say more.

**Comment by tony-matheus** [comment]
@Jairwin-L if you are using `next` (what looks like you do), you may try to dynamically import the component with `{ ssr: false}`

**Comment by johndavemanuel** [comment]
@Jairwin-L did you able to make it work on node 14?

**Comment by Jairwin-L** [comment]
> @Jairwin-L did you able to make it work on node 14?

sorry, I kind of forgot about it
